index.bs

```diff
@@ -568,6 +571,22 @@ through a process called [=sensor fusion=].
 [=calibration|Calibrated=] [=raw sensor readings=] are referred to as <dfn>sensor readings</dfn>,
 whether or not they have undergone [=sensor fusion=].
 
+The term <dfn id="concept-platform-sensor">platform sensor</dfn> refers to an implementation,
```

Comment:
maybe replace "implementation"?

implementation => "platform API" or "platform interface used by the UA"

Reply:
done